src/org/ods/OdsPipeline.groovy

```diff
@@ -18,12 +18,15 @@ class OdsPipeline implements Serializable {
   // Main entry point.
   def execute(Closure stages) {
     logger.info "***** Starting ODS Pipeline *****"
+    prepareForMultiRepoBuild()
```

Comment:
For better readability, I suggest to have the guarding condition around the method and not inside. Example:

```
if (!!script.env.MULTI_REPO_BUILD) {
    prepareForMultiRepoBuild()
}
```